io/net/ethtool/ethtool_tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+# Tests the network driver and interface with 'ethtool' command.
+# 12 different options of ethtool command are tested for each
+# of the interfaces specified in 'config' file
+
+PATH=$(avocado "exec-path"):$PATH
+
+# Install dependencies
+if [[ `python -c 'from avocado.utils.software_manager import SoftwareManager; \
+    print SoftwareManager().install("ethtool")'` == 'False' ]]
+then
+    avocado_debug 'Ethtool not installed'
+    exit
+fi
+
+CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
+interfaces=$(cat $CONFIG_FILE | grep -w "Interfaces" | awk -F'=' '{print $(NF)}')
+echo $interfaces
```

Comment:
I'd include some extra info in this echo. With only one interface in config file, this will echo the same information as in line 36 below. Something like `Device list: eth0 eth1`. And in line 36, `Testing device eth0`...